src/webhooks/pubsub/index.ts

```diff
@@ -19,7 +19,7 @@ import { db } from '@utils/db';
 
 const DEFAULT_REPOS = [SENTRY_REPO];
 const GH_API_PER_PAGE = 100;
-const DEFAULT_TEAM_LABEL = 'Team: Open Source';
+const DEFAULT_PRODUCT_AREA_LABEL = 'Product Area: Open Source';
```

Comment:
Should this still be the default product area label since `Product Area: Open Source` doesn't exist?

Reply:
Good catch, done in 934ea3b57cf731ba7259b72bdcc2b300f96c7fb4.